packages/flutter/test/widget/input_test.dart

```diff
@@ -633,7 +633,11 @@ void main() {
     TestGesture gesture = await tester.startGesture(firstPos, pointer: 7);
     await tester.pump();
     await gesture.moveBy(new Offset(0.0, -1000.0));
-    await tester.pump();
+    await tester.pump(const Duration(seconds: 2));
+    // Wait and drag again to trigger https://github.com/flutter/flutter/issues/6329
+    // (No idea why this is necessary, but the bug wouldn't repro without it.)
```

Comment:
Can one can put a print somewhere in the code (even today's code) that shows a difference between the test and manual testing?